minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -241,14 +241,8 @@ protected void setupTest() throws IOException {
     // If we are using the LocalMiniDFSCluster or S3UnderStorageCluster or OSSUnderStorageCluster,
     // we need to update the UNDERFS_ADDRESS to point to the cluster's current address.
     // This must happen after UFS is started with UnderFileSystemCluster.get().
-    // TODO(andrew): Move logic to the alluxio-tests module so that we can use instanceof here
-    // instead of comparing classnames.
-    if (mUfsCluster.getClass().getSimpleName().equals("LocalMiniDFSCluster")
-        || mUfsCluster.getClass().getSimpleName().equals("S3UnderStorageCluster")
-        || mUfsCluster.getClass().getSimpleName().equals("OSSUnderStorageCluster")) {
-      String ufsAddress = mUfsCluster.getUnderFilesystemAddress() + mHome;
-      Configuration.set(Constants.UNDERFS_ADDRESS, ufsAddress);
-    }
+    String ufsAddress = mUfsCluster.getUnderFilesystemAddress() + mHome;
+    Configuration.set(Constants.UNDERFS_ADDRESS, ufsAddress);
```

Comment:
In the new code, the configuration is always set to a new ufsAddress. I'm wondering what was it before this new value?